examples/Atlas_Database_Users/.gitignore

```diff
@@ -0,0 +1,4 @@
+atlaspl.code-workspace
+terraform.tfvars
+.terraform/
+*.tfstate*
```

Comment:
In the future (and if you have time today before I merge a commit w/ this would be good) be sure to run anything via a linter to check for common issues, such as this warning for no newline at end.

Reply:
Done